FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -283,4 +322,24 @@ NS_SWIFT_NAME(RemoteConfig)
 ///                         nil if the key doesn't exist in the default config.
 - (nullable FIRRemoteConfigValue *)defaultValueForKey:(nullable NSString *)key;
 
+#pragma mark - Realtime
+
+/// Completion handler invoked by fetch methods when they get a response from the server.
+///
+/// @param configUpdate Information on which key's values have changed
+/// @param error  Error message on failure.
+typedef void (^FIRRemoteConfigUpdateCompletion)(FIRRemoteConfigUpdate *_Nullable configUpdate,
+                                                NSError *_Nullable error)
+    NS_SWIFT_UNAVAILABLE("Use Swift's closure syntax instead.");
+
+/// Adds a listener to Realtime config updates. If it's the first listener, it starts off the
+/// stream.
+///
+/// @param listener              The configured listener that is called for every config update.
+/// @return                 Returns a registration that is used to remove a listener. If it is the
```

Comment:
Returns a registration representing the listener. The registration contains a `remove` method, which can be used to stop receiving for updates for this particular registration.